idea-plugin/src/main/kotlin/arrow/meta/plugin/idea/plugins/higherkinds/HigherKindsIdePlugin.kt

```diff
@@ -20,7 +20,7 @@ val IdeMetaPlugin.higherKindsIdePlugin: Plugin
     meta(
       addLineMarkerProvider(
         icon = ArrowIcons.HKT,
-        matchOn = { it.safeAs<KtClass>()?.takeIf(::isHigherKindedType) },
+        transform = { it.safeAs<KtClass>()?.takeIf(::isHigherKindedType) },
```

Comment:
With the current implementation this adds a line marker for the first child of a class.
- this child is not necessarily a leaf
- this child is not necessarily on the same line as the class name identifier (e.g. with annotations)